repositories/xp_boost_repository.py

```
@@ -0,0 +1,126 @@
+        return XpBoostModel.from_row(result[0]) if result else None
+
+    @staticmethod
+    async def get_boosts_for_target(
+        guild_id: str,
+        entity_ids: list[str],
+        target: BoostTarget = BoostTarget.ROLE,
+    ) -> list[XpBoostModel]:
+        """Get all boost entries for a list of entity IDs under a target type."""
+        if not entity_ids:
+            return []
+        from api import execute_query_iter
```

Flake8 (7.3.0)

[error] 'api.execute_query_iter' imported but unused

(F401)